### PR TITLE
some refine for pipeline

### PR DESCRIPTION
**Pull request by SeaRise**
### What problem does this PR solve?

Issue Number: ref #6233

Problem Summary:

### Check List

Tests 

- [ ] Unit test
- [ ] Integration test
- [ ] Manual test (add detailed scripts or steps below)
- [ ] No code

Side effects

- [ ] Performance regression: Consumes more CPU
- [ ] Performance regression: Consumes more Memory
- [ ] Breaking backward compatibility

Documentation

- [ ] Affects user behaviors
- [ ] Contains syntax changes
- [ ] Contains variable changes
- [ ] Contains experimental features
- [ ] Changes MySQL compatibility

### Release note

```release-note
None
```

### CODE REVIEW

File: dbms/src/Common/CPUAffinityManager.h [hunk 1]
```diff
@@ -52,7 +52,7 @@ struct CPUAffinityConfig
     // cop-pool and batch-cop-pool are the thread name of thread-pool that handle coprocessor request.
     // grpcpp_sync_ser is the thread name of grpc sync request thread-pool. However, this thread-pool is resize dynamically and we set these threads' cpu affinity in FlashService for simplicity.
     // Query threads of MPP tasks are created dynamiccally and we set these threads' cpu affinity when they are created.
-    std::vector<std::string> query_threads = {"cop-pool", "batch-cop-pool", "grpcpp_sync_ser"};
+    std::vector<std::string> query_threads = {"grpcpp_sync_ser"};
```
Comment (SeaRise) [inline dbms/src/Common/CPUAffinityManager.h:55]: cop-pool-thread-pool and batch-cop-thread-pool has been removed.

**Review by windtalker — APPROVED** [review]
LGTM

**Review by guo-shaoge — APPROVED** [review]

**Pull request merged**